### PR TITLE
Added displayMode 'first-image' and 'showThumbnails' prop

### PR DESCRIPTION
**Pull request by claudiu-iviteb**
#### What problem is this solving?

On the product-images app, you cand choose to show only one image instead of just carousel and list. And you can also choose to hide thumbnails when carousel is selected.

#### How to test it?

On the link below, you cand see the app linked and using the setting "showThumbnails": false
https://storecomponents--iviteb.myvtex.com/playstation-5/p

[Workspace](https://storecomponents--iviteb.myvtex.com/playstation-5/p)

#### Screenshots or example usage:

image
image

### CODE REVIEW

File: docs/ProductImages.md [hunk 1]
```diff
@@ -47,14 +47,15 @@
 | `position`                | `Enum`    | Set the position of the thumbnails (`left` or `right`). Only used when `thumbnailsOrientation` is `vertical` | `left`        |
 | `showNavigationArrows`             | `boolean`                                   | Controls if the navigation arrows should appear | `true`          |
 | `showPaginationDots`             | `boolean`                                   | Controls if the pagination dots should appear | `true`          |
+| `showThumbnails`             | `boolean`                                   | Controls if the thumbnails should appear in `carousel` displayMode | `true`          |
```
Comment (icazevedo) [inline docs/ProductImages.md:50]:
```suggestion
| `thumbnailVisibility`             | `visible` or `hidden`                                   | Controls if the thumbnails should appear in `carousel` displayMode | `visible`          |
```

Avoid boolean props

File: react/components/ProductImages/index.js [hunk 2]
```diff
@@ -109,6 +110,21 @@ const ProductImages = ({
       </div>
     )
 
+  else if (displayMode === DISPLAY_MODE.FIRST_IMAGE)
```
Comment (icazevedo) [inline react/components/ProductImages/index.js:113]: Add keys to the if clauses and check if there's any image before rendering the `[0]` position.

**Review by icazevedo — CHANGES_REQUESTED** [review]
Thanks for contributing! Please take a look at the suggestions I've made and fix the lint errors.

**Review by icazevedo — CHANGES_REQUESTED** [review]
Fix the lint errors and I think it'll be good to go :)

**Review by icazevedo — CHANGES_REQUESTED** [review]
Also, it seems that you just changed the docs from `showThumbnail` to `thumbnailVisibility`, not the actual implementation. Could you finish that?

**Review by klzns — CHANGES_REQUESTED** [review]
Do you mind allowing edits from us?

https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork

So I can rebase your branch

**Comment by claudiu-iviteb** [comment]
I don't have that checkmark like in the article.
image

File: react/components/ProductImages/index.js [hunk 3]
```diff
@@ -108,6 +110,25 @@ const ProductImages = ({
         ))}
       </div>
     )
+  }
+
+  if (displayMode === DISPLAY_MODE.FIRST_IMAGE && images.length) {
```
Comment (lbebber) [inline react/components/ProductImages/index.js:115]:
```suggestion
  if (displayMode === DISPLAY_MODE.FIRST_IMAGE && images?.length) {
```

**Review by lbebber — CHANGES_REQUESTED** [review]

**Review by igorbrasileiro — APPROVED** [review]

**Review by icazevedo — APPROVED** [review]

**Comment by vtexdaniel** [comment]
@igorbrasileiro @icazevedo can this be merged please or do we need additional approvals? 

Thanks!

**Review by brunoamui — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 4]
```diff
@@ -15,7 +18,6 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [3.143.1] - 2021-04-28
 ### Fixed
 - Makes main SearchBar placeholder translatable
```
Comment (victorhmp) [inline CHANGELOG.md:20]: Please revert this change

**Review by victorhmp — APPROVED** [review]

File: CHANGELOG.md (hunk 4 above)
Comment (claudiu-iviteb) [reply CHANGELOG.md:70]: I'm not sure what do you mean. I've updated the changelog. Hope that's it :)

Comment (victorhmp) [reply CHANGELOG.md:70]: WIth the rebase things moved around, but you removed an empty line right below this one. Please put it back :)

**Review by victorhmp — APPROVED** [review]
The code LGTM! Please just revert that change in the CHANGELOG :)

**Review by igorbrasileiro — APPROVED** [review]

File: CHANGELOG.md (hunk 4 above)
Comment (claudiu-iviteb) [reply CHANGELOG.md]: Oh...yes, my bad. Thank you!

**Review by victorhmp — APPROVED** [review]
